Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/TestCertificateContainer.kt

```diff
@@ -52,7 +51,7 @@ abstract class TestCertificateContainer {
         }
 
     fun toTestCertificate(
-        valueSet: VaccinationValueSet?,
+        valueSet: VaccinationValueSets?,
```

Comment:
I think we need to inject  `valueSetsRepository` in this repo and provide the `latestVaccinationValueSets`

Reply:
I don't know what you exactly mean here..
I simply adjusted it to the new tech spec.

Reply:
I know , do you know if someone else will adapt this in another PR ?
maybe @d4rken ?

Reply:
What I meant  is `toTestCertificate` requires value sets which is implemented in this PR , similar to vaccinated person certificates this logic is handled in the repo , so we don't need to do this in every view model using the test Certificates

Reply:
I can adjust that in separate PR.